.gitignore

```diff
@@ -12,3 +12,4 @@ packages/opds-web-client/docs
 
 # Temporary files
 *~
+node_modules
```

Comment:
This should not be necessary (but doesn't hurt) because the root level `package.json` doesn't install any packages.

Reply:
Ah yes, it's actually part of my `prettier` PR where this becomes necessary